ui/jobs/utils.ts

```diff
@@ -94,66 +127,80 @@ export function calcGCDFromStat(bars, stat, actionDelay) {
   // TODO: this probably isn't useful to track
   const astralUmbralMod = 100;
 
-  const gcdMs = Math.floor(1000 - Math.floor(130 * (stat - kLevelMod[bars.level][0]) /
-    kLevelMod[bars.level][1])) * actionDelay / 1000;
+  const gcdMs = Math.floor(1000 - Math.floor(
+      130 * (stat - kLevelMod[bars.level][0]) / kLevelMod[bars.level][1],
+  )) * actionDelay / 1000;
   const a = (100 - type1Buffs) / 100;
   const b = (100 - type2Buffs) / 100;
   const gcdC = Math.floor(Math.floor((a * b) * gcdMs / 10) * astralUmbralMod / 100);
   return gcdC / 100;
-}
+};
 
-export function computeBackgroundColorFrom(element, classList) {
+export const computeBackgroundColorFrom = (element: HTMLElement, classList: string):string => {
   const div = document.createElement('div');
-  const classes = classList.split('.');
-  for (let i = 0; i < classes.length; ++i)
-    div.classList.add(classes[i]);
+  classList.split('.').forEach((item) => {
+    div.classList.add(item);
+  });
   element.appendChild(div);
   const color = window.getComputedStyle(div).backgroundColor;
   element.removeChild(div);
   return color;
-}
-
-export function makeAuraTimerIcon(name, seconds, opacity, iconWidth, iconHeight, iconText,
-    barHeight, textHeight, textColor, borderSize, borderColor, barColor, auraIcon) {
-  const div = document.createElement('div');
-  div.style.opacity = opacity;
-
-  const icon = document.createElement('timer-icon');
-  icon.width = iconWidth;
-  icon.height = iconHeight;
-  icon.bordersize = borderSize;
+};
+
+export const makeAuraTimerIcon = (
+    name: string,
+    seconds: number,
+    opacity: number,
+    iconWidth: number,
+    iconHeight: number,
+    iconText: string,
+    barHeight: number,
+    textHeight: number,
+    textColor: string,
+    borderSize: number,
+    borderColor: string,
+    barColor: string,
+    auraIcon: string,
+): HTMLDivElement => {
+  const div: HTMLDivElement = document.createElement('div');
+  div.style.opacity = opacity.toString();
+
+  const icon = document.createElement('timer-icon') as TimerIcon;
+  icon.width = iconWidth.toString();
+  icon.height = iconHeight.toString();
+  icon.bordersize = borderSize.toString();
   icon.textcolor = textColor;
   div.appendChild(icon);
 
-  const barDiv = document.createElement('div');
+  const barDiv: HTMLDivElement = document.createElement('div');
```

Comment:
```suggestion
  const barDiv = document.createElement('div');
```

doesn't need a type annotation here 'cause TypeScript knows that `createElement` returns a `HTMLDivElement`.

here and everywhere.